### PR TITLE
[TH-344] Add step page layout, step 1, 2 UI

### PR DESCRIPTION
**Pull request by ssk7808**
# Description
- campaign step 페이지 레이아웃, step 1, 2 UI 구현입니다. step 2 에서 create futurepass 부분은 구현해야합니다.
- 지갑 연결 로딩/연결완료 ui는 그려만 놨고, 추후 api 연결 필요합니다.

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration

- [ ] Test A : isLoading, isConnected 값을 true, false 로 바꿔가며 테스트

## Screenshots (if appropriate):
스크린샷 2023-11-15 오후 4 31 46

https://github.com/TeamHeimdallr/moai-web/assets/29878878/f08c3126-7684-4fdf-83f0-f25c7c8878b0

### CODE REVIEW

File: src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+          <FpassTextWrapper>
+            Futurepass
+            <ButtonIconSmall
+              data-tooltip-id={TOOLTIP_ID.CAMPAIGN_FUTUREPASS}
+              icon={<IconQuestion width={16} height={16} />}
+            />
+          </FpassTextWrapper>
+        </TitleWrapper>
+      )}
+      <WalletWrapper>
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx]: [apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+              </WalletInnerWrapper>
+              {isLoading && <LoadingIcon src={imageStepLoading} width={24} height={24} />}
+              {isConnected && (
+                <ConnectedWrapper>
+                  <ConnetedDot />
+                  Connected
+                </ConnectedWrapper>
+              )}
+            </Wallet>
+          ) : null
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx]: [apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+            <ButtonIconSmall
+              data-tooltip-id={TOOLTIP_ID.CAMPAIGN_FUTUREPASS}
+              icon={<IconQuestion width={16} height={16} />}
+            />
+          </FpassTextWrapper>
+        </TitleWrapper>
+      )}
+      <WalletWrapper>
+        {wallets.map(w =>
+          w.type === chain ? (
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx]: [apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+            <Wallet
+              key={w.name}
+              onClick={() => {
+                w.onClick();
+                close();
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx]: [apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/pages/step/index.tsx [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+        </MoveStepWrapper>
+        <TextWrapper>
+          <StepText>Step {step}</StepText>
+          <Title>{titleText}</Title>
+        </TextWrapper>
+        {step === 1 ? (
+          <StepConnectWallet step={1} />
+        ) : step === 2 ? (
+          <StepConnectWallet step={2} />
+        ) : null}
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/pages/step/index.tsx]: [apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/states/step.ts [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+/**
+ * swap histories table soring state in pool detail page
+ */
+export const useCampaignStepStore = create<State>()(
+  immer(
+    logger(set => ({
+      step: 1,
+      setStep: (step: number) => set({ step }),
```
[the remark this answers is not in the document]
Comment (ssk7808) [reply src/pages/campaign/states/step.ts]: 좋은 리뷰 감사합니다

[apply code review](https://github.com/TeamHeimdallr/moai-web/pull/191/commits/be33ac9187152078a66bbde0f2dad25f9b0de0bb)

File: src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (jihyune) [reply src/pages/campaign/components/connect-wallet/step-connect-wallet.tsx]: 수정안됐습니당..

**Review by jihyune — APPROVED** [review]
👍

**Pull request merged**